.github/workflows/integration-tests.yml

```diff
@@ -311,6 +315,91 @@ jobs:
           else
             echo "run_command=./smoke/${{ matrix.product.name }}_test.go" >> "$GITHUB_OUTPUT"
           fi
+      - name: Check for "enable tracing" label
+        id: check-label
+        run: |
+          label=$(jq -r '.pull_request.labels[].name' "$GITHUB_EVENT_PATH")
+          if [[ "$label" == "enable tracing" ]]; then
```

Comment:
the space in "enable tracing" looks suspicious as a label. i expected a - or _

Reply:
Good eye - It's a little funky, but a valid label name. We use both `-` and ' ' for our current labels